new/server-stats-endpoint.md

```diff
@@ -0,0 +1,115 @@
+---
+RFC: unassigned
+Title: Stats Endpoint for Chef Server
+Author: Jay Mundrawala <jmundrawala@chef.io>
+Status: Draft
+Type: <Standards Track, Informational, Process>
+<Replaces: RFCxxx>
+<Tracking:>
+<  - https://github.com/chef/chef/issues/X>
+---
+
+# Stats Endpoint for Chef Server
+
+Monitoring Chef Server is an involved task. There are several services that make
+up a Chef Server, all of which need to be monitored. These services each have their
+own set of metrics, and each has it's own way obtaining those metrics. The Chef Server
+component itself has 2 ways that provide a different set of metrics: pushing to statsd 
+and through Folsom, which can be set up to push to Graphite.
+
+This RFC introduces a stats endpoint to Chef Server which is capable of providing metrics
+for Chef Server and all its co-located services.
+
+
+## Motivation
+
+    As a Chef Server operator,
+    I want to have a consistent way to get important Chef Server metrics,
+    so that I can easily monitor the health of my Chef Server out of the box.
+
+    As a Chef Server operator,
+    I want to understand the metrics I'm given,
+    so that I can easily monitor the health of my Chef Server.
+
+
+## Specification
+
+This RFC adds a series of `/_stats/$service` endpoints to Chef Server. Each service
+colocated on the box will be provided an endpoint, and with respond to GET requests.
+The response format will be a text format which is specified by 
+[Prometheus 0.0.4](https://prometheus.io/docs/instrumenting/exposition_formats/#text-format-details).
```

Comment:
We might want to address the fact that prometheus also has a binary/protobuf format and explain why we do or don't want to support it.